users/postgres/clients.go

```diff
@@ -5,78 +5,79 @@ package postgres
 
 import (
 	"context"
+	"database/sql"
+	"encoding/json"
 	"fmt"
+	"strings"
+	"time"
 
+	"github.com/absmach/magistrala/internal/api"
 	mgclients "github.com/absmach/magistrala/pkg/clients"
-	pgclients "github.com/absmach/magistrala/pkg/clients/postgres"
 	"github.com/absmach/magistrala/pkg/errors"
 	repoerr "github.com/absmach/magistrala/pkg/errors/repository"
+	"github.com/absmach/magistrala/pkg/groups"
 	"github.com/absmach/magistrala/pkg/postgres"
+	"github.com/absmach/magistrala/users"
+	"github.com/absmach/magistrala/users/storage"
+	"github.com/jackc/pgtype"
 )
 
-var _ mgclients.Repository = (*clientRepo)(nil)
-
-type clientRepo struct {
-	pgclients.Repository
+type userRepo struct {
+	Repository users.UserRepository
+	gcStorage  storage.Storage
```

Comment:
Let's not take care of profile photo upload here. We will treat it as a URL. However, we will need this in our UI to first upload photo and then update/create user (also, we don't want to upload photo twice - first to bakcend then to Cloud storage).